spec/presenters/candidate_interface/application_form_presenter_spec.rb

```diff
@@ -75,6 +75,40 @@
     end
   end
 
+  describe '#other_qualifications_completed?' do
+    it 'returns true if other qualifications section is completed' do
+      application_form = FactoryBot.build(:application_form, other_qualifications_completed: true)
+      presenter = CandidateInterface::ApplicationFormPresenter.new(application_form)
+
+      expect(presenter).to be_other_qualifications_completed
+    end
+
+    it 'returns false if other qualifications section is incomplete' do
+      application_form = FactoryBot.build(:application_form, other_qualifications_completed: false)
+      presenter = CandidateInterface::ApplicationFormPresenter.new(application_form)
+
+      expect(presenter).not_to be_other_qualifications_completed
```

Comment:
as before

Reply:
Links to https://github.com/DFE-Digital/apply-for-postgraduate-teacher-training/pull/459#discussion_r344088105